### PR TITLE
Google Custom Search Results Overrides

### PR DESCRIPTION
**Pull request by martypowell**
Override the default styles of google custom search.

### CODE REVIEW

**Comment by martypowell** [comment]
@baltimorecounty/front-end-developers looks like the forms styles are in this, so let me talk to @theilman7 about getting that in first before reviewing.

File: src/styles/overrides/_google-search-results.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    margin-bottom: $margin-small;
+  }
+
+  /* Description Text */
+  .gs-snippet {
+    font-family: $default-body-font !important;
+    font-size: $base-font-size;
+    /* Search Term*/
+    b {
+      font-weight: 600;
```
Comment (theilman7) [inline src/styles/overrides/_google-search-results.scss]: This should be updated with the variable `$font-medium`.

File: src/styles/overrides/_google-search-results.scss [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  &:disabled,
+  &.disabled {
+    color: $pale-blue-lightest !important;
+  }
+
+  /* current page of results link */
+  &.gsc-cursor-current-page {
+    color: $black !important;
+    font-weight: 900;
```
Comment (theilman7) [inline src/styles/overrides/_google-search-results.scss]: Do we want to add 900 to the font weight variables file? If not, this is fine as-is.

File: src/styles/overrides/_se-form.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      }
+
+      .seLabelCell {
+        @extend .dg_label;
+        margin-top: 0;
+        margin-bottom: 5px;
+
+        .seRequiredMarker {
+          margin-left: 5px;
+        }
```
Comment (theilman7) [inline src/styles/overrides/_se-form.scss]: Recommend updating to `$base-spacing-unit`.

File: src/styles/overrides/_se-form.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      }
+
+      .seCheckbox {
+        @extend .dg_checkbox-input;
+      }
+
+      .seLabelCell {
+        @extend .dg_label;
+        margin-top: 0;
+        margin-bottom: 5px;
```
Comment (theilman7) [inline src/styles/overrides/_se-form.scss]: Recommend updating to `$base-spacing-unit`.

File: src/styles/overrides/_se-form.scss [hunk 5]
```diff
@@ -0,0 +1,92 @@
+.seform {
```
Comment (theilman7) [inline src/styles/overrides/_se-form.scss]: Did we already go over this file in a previous PR for form overrides?

**Comment by theilman7** [comment]
Branch needs to be updated.

**Comment by theilman7** [comment]
Branch needs to be updated to resolve conflicts.

**Comment by martypowell** [comment]
> Branch needs to be updated.

@theilman7 the branch conflicts have been resolved.

File: src/styles/overrides/_google-search-results.scss (hunk 2 above)
Comment (martypowell) [reply src/styles/overrides/_google-search-results.scss]: updated, added as `$font-boldest` as the new variable.

File: src/styles/overrides/_google-search-results.scss (hunk 1 above)
Comment (martypowell) [reply src/styles/overrides/_google-search-results.scss]: adjusted

File: src/styles/overrides/_google-search-results.scss [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+  padding: $default-padding !important;
+  margin-bottom: $default-margin !important;
+
+  /* Search result title (it is a link) */
+  .gs-title {
+    @extend h3;
+
+    text-transform: uppercase;
+    font-size: $h2-font-size !important;
+    font-weight: 900;
```
Comment (martypowell) [inline src/styles/overrides/_google-search-results.scss]: @theilman7 also changed this to $font-boldest

File: src/styles/overrides/_se-form.scss (hunk 4 above)
Comment (martypowell) [reply src/styles/overrides/_se-form.scss]: @theilman7 yes, it was a result of not merging, this file is no longer available.

File: src/styles/overrides/_google-search-results.scss (hunk 6 above)
Comment (theilman7) [reply src/styles/overrides/_google-search-results.scss]: That works.

**Review by theilman7 — DISMISSED** [review]
Approved.

**Pull request merged**